app/cli/internal/action/org_update.go

```diff
@@ -30,14 +30,24 @@ func NewOrgUpdate(cfg *ActionsOpts) *OrgUpdate {
 }
 
 type NewOrgUpdateOpts struct {
-	BlockOnPolicyViolation *bool
+	BlockOnPolicyViolation   *bool
+	PoliciesAllowedHostnames *[]string
 }
 
 func (action *OrgUpdate) Run(ctx context.Context, name string, opts *NewOrgUpdateOpts) (*OrgItem, error) {
 	client := pb.NewOrganizationServiceClient(action.cfg.CPConnection)
-	resp, err := client.Update(ctx, &pb.OrganizationServiceUpdateRequest{
-		Name: name, BlockOnPolicyViolation: opts.BlockOnPolicyViolation,
-	})
+
+	payload := &pb.OrganizationServiceUpdateRequest{
+		Name:                   name,
+		BlockOnPolicyViolation: opts.BlockOnPolicyViolation,
+	}
+
+	if opts.PoliciesAllowedHostnames != nil {
+		payload.PoliciesAllowedHostnames = *opts.PoliciesAllowedHostnames
```

Comment:
This is what I mean, `*variable` shouldn't be needed for slices, since they are pointers anyways.

Reply:
this is about making sure we can detect when we want to update the value vs empty it. In any case I've implemented the same by using slices